### PR TITLE
book: add initial book

### PR DESCRIPTION
**Pull request by ahayzen-kdab**
Related to #23 

Change-Id: I332c3c172a5b5c928884d8438309eb3fdae992dd

### CODE REVIEW

File: book/src/concepts/qt.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<!--
+SPDX-FileCopyrightText: 2021 Klarälvdalens Datakonsult AB, a KDAB Group company <info@kdab.com>
+SPDX-FileContributor: Andrew Hayzen <andrew.hayzen@kdab.com>
+
+SPDX-License-Identifier: MIT OR Apache-2.0
+-->
+
+# Qt
+
+## Inovkables
```
Comment (dfaure-kdab) [inline book/src/concepts/qt.md:10]: Typo: Invokables

Comment (ahayzen-kdab) [reply book/src/concepts/qt.md]: Nice spot, thanks!

**Comment by ahayzen-kdab** [comment]
This now needs #81 to land first.

**Review by LeonMatthesKDAB — APPROVED** [review]
Let's get this initial version merged now and optimize later.

**Pull request merged**